src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -405,9 +409,9 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     // reset name request
     this.setNameRequest({
       legalType: this.getEntitySnapshot?.businessInfo?.legalType,
-      legalName: this.getEntitySnapshot?.businessInfo?.legalName,
-      nrNumber: this.getEntitySnapshot?.businessInfo?.nrNumber
-    })
+      nrNum: this.getEntitySnapshot?.businessInfo?.nrNumber
+    } as any)
+    this.setNameRequestLegalName(this.getEntitySnapshot?.businessInfo?.legalName)
```

Comment:
Reminder to myself: Need to change this in my PR as `this.getEntitySnapshot?.businessInfo` is changed to `getOriginalBusinessInfo`.